README.md

```diff
@@ -2,13 +2,77 @@
+
+![model](https://docs.google.com/drawings/d/1ZmlgstkpoOeUrdeCkWn7PziXC8iqS3eh9281bjTyxX4/pub?w=960&h=720)
+
+### [Agent](https://github.com/valueflows/agent)
+
+An `Agent` is a thing who can act: as in a person, group, or bot.
+
+> Example: "an example of an agent is me, Mikey (@ahdinosaur)! i have a name, an email, etc. each agent, like everything in Value Flows, has an associated type object. for agents, these can correspond to the [different type of agents in Open OS](https://enspiral.gitbooks.io/open-os/content/en/levels_of_organising.html), namely Person, Pod, Community, and Network. on the type we can specify properties of that level of organizing structure."
+Agents are **fractal identities**
+
+### [Resource](https://github.com/valueflows/resource)
+
+A `Resource` is a thing that can be acted on.
+
+> Example: "a resource is a physical thing, a resource type is the abstract ideal of that thing. when you go on Amazon you browse through resource types you can buy, the physical thing that you receive in the mail is a resource. when you go to the library catalog you browse through types of books, which correspond to physical books you can check out. these resource types can relate to each other as a taxonomy, as we see with categories on Amazon and genres in the library."
+
+### Action
+
+An `Action` is how an agent can act with resource(s).
+
+- **Transfer**: [a change in 'rights' relating to the resource(s)](https://github.com/valueflows/exchange/blob/master/Transfer.md)
+- **Transport**: [a change in location of the resource(s)](https://github.com/valueflows/process/blob/master/Transportation.md)
+- **Transform**: [a change in the composition of the resource(s)](https://github.com/valueflows/process)
+- ... possibly more?
+
+> Examples:
+>
+> - "a transfer is when a resource changes hands. for example, i give you three dollars in exchange for a coffee. the coffee and money resources are still the same, but they've changed owners."
+> - "a transform is when a resource changes composition. for example, i input the resources of coffee grounds and hot water into a coffee machine resource in order to output hot coffee. the hot water and electricity are consumed during use, the coffee grounds are turned into compostable waste, and the coffee machine is used but still available for more coffee."
+
+A re-usable action type is also known as a **Recipe**.
+
+Actions describe _flows of resources_, as opposed to the more common double-entry accounting. It's possible to derive double-entry accounting from a resource flow, but not the other way around.
+
+### Signal
+
+A `Signal` is what an agent can say about an action, often in a ['conversation'](http://conversationsforaction.com/cfa-playground).
+
+- **Intend**: [express inputs for action](https://github.com/valueflows/intent)
+- **Plan**: express commitment (promise, schedule) to follow plan
+- **Observation**: account for observed reality of action
+- **Report**: retrospect on outcome of action
```

Comment:
Could you maybe illustrate it based on one of existing use cases or add new use case if needed?